go/master/client.go

```diff
@@ -20,6 +25,51 @@ type Client struct {
 	ch   chan []byte
 }
 
+// MasterAddresser provide master address
+type masterAddresser struct {
+	client    *clientv3.Client
+	timeout   time.Duration
+	endpoints []string
+}
+
+// Address return the address
+func (m masterAddresser) Address() string {
+	for {
+		ctx, cancel := context.WithTimeout(context.Background(), m.timeout)
+		resp, err := m.client.Get(ctx, masterAddrPath)
```

Comment:
@Yancey1989 Thanks!
About the unit test, for example, the master server is taking an interface `Store` as the parameter: https://github.com/PaddlePaddle/Paddle/blob/develop/go/master/service.go#L94.
The definition of store is:
```Go
type Store interface {
	Save([]byte) error
	Load() ([]byte, error)
}
```
We have an implementation of `Store` using etcd. But to test the master server, it's easy, we don't to run a etcd cluster. We can just write a stub interface for `Store`. However this only tests master service. Does not test the etcd `Store` implementation. But that's a good start.